incubator/swift/image/Dockerfile-stack

```diff
@@ -1,6 +1,10 @@
 FROM swift:5.0
 
-RUN apt-get update && apt-get install -y libcurl4-openssl-dev libssl-dev
+RUN apt-get update \
+ && apt-get dist-upgrade -y \
```

Comment:
What's the reason for `dist-upgrade`? Docker say not to do this: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Removed